tests/unit/swagger/definitionsGeneration/definitions.spec.ts

```diff
@@ -66,6 +66,7 @@ describe('Definition generation', () => {
 
   describe('Interface-based generation', () => {
     it('should generate a definition for referenced models', () => {
+      console.log(allSpecs[0].spec.definitions);
```

Comment:
I believe we should instead add a test similar to the OpenAPI 3 one (using `getDefinition('...')`) and remove the log.

Reply:
Fixed and added.

Reply:
Also without the `console.log` this time ;)